py/server/deephaven/table.py

```diff
@@ -1949,6 +1980,70 @@ def format_row_where(self, cond: str, formula: str) -> Table:
         except Exception as e:
             raise DHError(e, "failed to color format rows conditionally.") from e
 
+    def format_data_bar(self, column: str, value_column: str = None, min: float = None, max: float = None,
+                        axis: Union[AxisOption, str] = None, positive_color: Union[str, List[str]] = None,
+                        negative_color: Union[str, List[str]] = None, value_placement: Union[ValuePlacementOption, str] = None,
+                        direction: Union[DirectionOption, str] = None, opacity: float = None, marker_column: str = None,
+                        marker_color: str = None) -> Table:
+        """ Applies data bar formatting to the column specified.
+
+        Args:
+            column (str): where to place the data bars
```

Comment:
Arg strings somehow feel stylisticly inconsistent with the rest of the docs.  I can't say why.  Give the existing docs a quick scan and see if there are obvious changes.  @jmao-denver may also have comments.
e.g. "where to get the values to form data bars from" -> "column containing the values to form data bars from" may make the docs more consistent.

Reply:
The syntax is correct.  The grammar doesn't feel consistent with the other docs.  For example `where to place the data bars` feels inconsistent with what we have in `deephaven.table.Table`.  As an example https://deephaven.io/core/pydoc/code/deephaven.table.html#deephaven.table.PartitionedTable.sort has `the columns(s) to be sorted on.  Can't include the constituent column.`  If similar grammar is applied to `column` in this case, it would end up as something like `the column data bars are placed into`.

Reply:
Touched up the grammar for these. Let me know if it's still not quite up to par